listenbrainz/webserver/static/css/listens-page.less

```diff
@@ -310,7 +310,12 @@
 				padding-right:initial;
 			}
 		}
-		
+
+		.title-duration {
+			display: flex;
+			flex-direction: row;
```

Comment:
I would also add `align-items: center;` so that if the track title wraps around, the duration is vertically centered:

<img width="421" alt="image" src="https://user-images.githubusercontent.com/6179856/166910824-6d304455-389c-4b27-b6f7-3622bcf4d289.png">

Reply:
Noted.